src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResources.java

```diff
@@ -0,0 +1,114 @@
+/*
+RequireThis
+checkFields = (default)true
+checkMethods = (default)true
+validateOnlyOverlapping = (default)true
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.requirethis;
+
+import java.io.*;
+import java.util.Scanner;
+
+public class InputRequireThisTryWithResources implements AutoCloseable{
+    private BufferedReader bufferedReader;
+    private InputStreamReader streamReader;
+    private Scanner scanner;
```

Comment:
Please do me big favor and name Fields by some notation "fieldXxxxx" or "fldXxxx"or "mXxxx". To easily catch reason of violation.

Reply:
Done